common/src/main/java/io/seata/common/util/NetUtil.java

```diff
@@ -83,7 +83,7 @@ public static String toStringAddress(InetSocketAddress address) {
      * @return the inet socket address
      */
     public static InetSocketAddress toInetSocketAddress(String address) {
-        int i = address.indexOf(':');
+        int i = address.lastIndexOf(':');
```

Comment:
It seems to me that NetUtil's IPV6 support is not just about this.

`LOCALHOST`  `ANY_HOST`  `IP_PATTERN`  `toLong`